components/gcp/container/Dockerfile

```diff
@@ -12,19 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM python:2.7-slim-jessie
+FROM python:3.7-slim
 
-RUN apt-get update && apt-get install -y --no-install-recommends \
-	wget patch \
-	&& rm -rf /var/lib/apt/lists/*
-
-RUN pip2 install apache-beam[gcp]==2.10.0
+RUN pip install apache-beam[gcp]
 RUN pip install pandas
 
 ADD build /ml
 WORKDIR /ml
 RUN pip install .
 
-RUN patch /usr/local/lib/python2.7/site-packages/googleapiclient/http.py < /ml/patches/http.patch
```

Comment:
Umm....this is needed for telemetry tracking for KFP on other GCP services. Can we add this back?

Reply:
Thanks for the info. I restore thed patch and added some comments.
This isn't ideal as the patch is based on a specific version of the code. It complicates the package version upgrade. But the other option is quite costly. We should probably revisit this later when have bandwidth.